client/src/components/SignUp.js

```diff
@@ -141,37 +202,58 @@ function SignUp(props) {
             fullWidth
             required
             type="password"
-            error={!!errors.password}
-            helperText={errors.password}
+            // error={!!errors.password}
+            // helperText={errors.password}
+            inputProps={{ minLength: 6 }}
             onChange={(e) => setPassword(e.target.value)}
           />
           <label>Confirm Password:</label>
-          <TextField
-            className={classes.textField}
-            variant="outlined"
-            label="confirm password"
-            fullWidth
-            required
-            type="password"
-            error={!!errors.confirm}
-            helperText={errors.confirm}
-            onChange={(e) => setConfirm(e.target.value)}
-          />
+          <ErrorTooltip
+            open={openTooltip}
+            title={<Typography>Passwords must match.</Typography>}
+            arrow
+            disableHoverListener
+            disableTouchListener
+            disableFocusListener
+          >
+            <TextField
+              className={classes.textField}
+              variant="outlined"
+              label="confirm password"
+              fullWidth
+              required
+              type="password"
+              error={!!errors.confirm}
+              // helperText={errors.confirm}
+              onChange={(e) => setConfirm(e.target.value)}
+            />
+          </ErrorTooltip>
           <Button
             className={classes.button}
+            // onClick={handleClick}
```

Comment:
Be sure to remove this and all the other commented out code above before merging